README.md

```diff
@@ -1,2 +1,9 @@
 # MyWeb
```

Comment:
It would be great to see some instructions about how to run this project and what prerequisites are needed